src/lifecycle.ts

```diff
@@ -72,7 +71,7 @@ export class FirebaseFunctionsTest {
 
   /** Complete clean up tasks. */
   cleanup() {
-    forEach(this._oldEnv, (val, varName) => {
+    Object.entries(this._oldEnv).forEach(([varName, val]) => {
```

Comment:
nit: IIRC Google style guide says we should just be doing `for of` instead of `forEach`

Reply:
Done!